other/management/commands/clean-notifications.py

```diff
@@ -0,0 +1,23 @@
+"""Clean up old notifications (older than 3 months)"""
+from datetime import timedelta
+import json
+
+from django.utils.timezone import now
+from django.core.management.base import BaseCommand
+from notifications.models import Notification
+
+class Command(BaseCommand):
+    help = 'Clean up old notifications (older than 3 months)'
+
+    def add_arguments(self, parser):
+        parser.add_argument('path', type=str, help='Give directory path for dumping old notifs without end slash')
+
+    def handle(self, *args, **options):
+        file_path = options['path'] + '/' + str(now().date()) + '.txt'
+        queryset = Notification.objects.filter(timestamp__lte=now() - timedelta(days=90))
+        print('Dumping and Cleaning up %s old notifications' % queryset.count())
+        if queryset.count() > 0:
+            data = list(queryset.values())
+            with open(file_path) as f:
+                json.dump(data, f, ensure_ascii=False, default=str)
```

Comment:
This is not right. If you dump arrays of lists you can join files by simply appending them together. Also this whole thing is going to be a single line, which isn't cool for grep. Iterate the list and write each dict to the file instead.

Reply:
Also the `queryset.count` check is mostly irrelevant if you do this along with using a single file.

Reply:
Now the old notifs are written one per line to file